modules/monitoring-resizing-a-persistent-volume.adoc

```diff
@@ -27,6 +27,8 @@ No service disruption occurs during this process.
 
 .Procedure
 
+. Manually expand a PVC with the updated storage request. For more information, see "Expanding persistent volume claims (PVCs) with a file system" in _Expanding persistent volumes_.
```

Comment:
I followed these docs guidelines to include the link: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences

Reply:
Hello @machine424 here I only refer to the section, because, due to our guidelines, I cannot add a link directly. So the rule here is to include it as I did. The link can then be found in the additional resources section. Thank you!

Reply:
It's just not straightforward (I think) to find the `Expanding persistent volumes` section...